openlibrary/core/lending.py

```diff
@@ -52,8 +52,11 @@ def setup(config):
     if config.get("content_server"):
         try:
             config_content_server = ContentServer(config.get("content_server"))
-        except:
-            pass
+        except Exception:
```

Comment:
Disregard previous comment about logging -- this looks better (we still likely want to except more specific exceptions and consider the reprecutions when config_content_server is not set